aspnetcore/fundamentals/logging/index.md

```diff
@@ -1180,6 +1180,14 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 For more information, see each provider's documentation. Third-party logging providers aren't supported by Microsoft.
 
+## Third-party logging samples
```

Comment:
Not saying we'll accept this PR, buf if you do, you'd need to move this to `## Additional resources` and make it one line.

Reply:
Not sure this meets DR's cutoff for *value*. AFAIK IIRC, the official provider docs have Core samples and link to additional external resources. Also, we haven't looked at this, and *someone* should look. Official docs probably just can't link anything sent in without review.

@akovac35, this would be good for the Community Standup and https://github.com/quozd/awesome-dotnet.

Reply:
Thanks for reviewing.

I have updated the proposal as suggested by @Rick-Anderson. @guardrex: thanks for suggestions. The official docs have samples which cover only the very basics. The samples I prepared cover much, much more and that for the Microsoft.Extensions.Logging.ILogger interface in various scenarios.

Because of using third party resources as logger providers (NLog, Serilog) I could not merge them with your samples, so a new repository was created. I think the value provided for site users exceed reasonable thresholds, I really covered many details.